WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerceUITests.xcscheme

```diff
@@ -20,6 +20,20 @@
                ReferencedContainer = "container:WooCommerce.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "F997170123DBB97500592D8E"
+               BuildableName = "WooCommerceScreenshots.xctest"
+               BlueprintName = "WooCommerceScreenshots"
+               ReferencedContainer = "container:WooCommerce.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
This is the setting in the UI tests scheme that makes is build the screenshots automation tests , too. The rationale is that this way we'll know if the screenshots tests fail to build because of a change in the shared files sooner.